tests/exploration/test_devi_manager_spin.py

```
@@ -0,0 +1,121 @@
+import os
```

Ruff

`os` imported but unused

Remove unused import: `os`

(F401)